src/ui/Controllers/CourseController.cs

```diff
@@ -20,11 +20,10 @@ public CourseController(ManageApi manageApi)
         {
             _manageApi = manageApi;
         }
-
         [Route("{accreditingProviderId=self}/{courseTitle}/{ucasCode}")]
-        public async Task<IActionResult> Variants(string accreditingProviderId, string courseTitle, string ucasCode)
+        public async Task<IActionResult> Variants(string accreditingProviderId, string courseTitle, string ucasCode, string organisationId)
```

Comment:
What is the expect url in the browser?
@fofr 
`{accreditingProviderId=self}/{courseTitle}/{ucasCode}?organisationId=123`
site.com/course/self/coursetitle/ucascode/?organisationId=123

I think it should be
site.com/course/123/self/coursetitle/ucascode/
as we already have
site.com/course/123

Reply:
Url fixed as suggested